doc/nrf/releases/release-notes-latest.rst

```diff
@@ -200,22 +202,71 @@ Zigbee
 Common
 ======
 
+Trusted Firmware-M
+------------------
+
+* Added support for hardware-accelerated cryptography in TF-M using the Nordic Security module (nrf_security).
+  When enabled (default), any calls to psa_crypto APIs will utilize the CryptoCell hardware on nRF9160 and nRF5340.
+* Added support for using hardware unique keys (HUKs) for key derivation (``TFM_CRYPTO_ALG_HUK_DERIVATION``).
+  TF-M automatically generates and stores random hardware unique keys (if not present), using the :ref:`lib_hw_unique_key` library.
+* TFM_MINIMAL: Added a size-optimized configuration of TF-M which provides a minimal set of features:
+
+  * This configuration requires 32 kB of flash and provides random number generation, SHA-256, and the platform memory read service, which is analogous to the feature set of :ref:`secure_partition_manager`.
+  * The configuration is showcased in the :ref:`tfm_hello_world` sample.
+
```

Comment:
In the TF-M section we can add the changes for the protected storage service: 

The TF-M protected storage service is now using non-static labels when deriving encryption keys. The derivation labels are generated by combining the client id of the requesting service and the UID of the resource.

Reply:
@oyvindronningstad could you implement this change, please?